src/Sylius/Bundle/GridBundle/Form/Type/Filter/EntityFilterType.php

```diff
@@ -26,12 +26,18 @@
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $idOptions = [
+            'class' => $options['class'],
+            'label' => false,
+            'placeholder' => 'sylius.ui.all',
+        ];
+
+        if (null !== $options['choice_label']) {
+            $idOptions['choice_label'] = $options['choice_label'];
+        }
+
         $builder
-            ->add('id', EntityType::class, [
-                'class' => $options['class'],
-                'label' => false,
-                'placeholder' => 'sylius.ui.all',
-            ])
+            ->add('id', EntityType::class, $idOptions)
```

Comment:
Why can't we just set `EntityType` as its parent and then only overwrite `label` and `placeholder` options in `configureOptions`?